cppcache/integration-test/ThinClientFailover3.hpp

```diff
@@ -242,21 +242,22 @@ void doNetsearch(const char* name, const char* key, const char* value) {
   LOG("Netsearch complete.");
 }
 
-std::vector<char*> storeEndPoints(const char* points) {
-  std::vector<char*> endpointNames;
-  if (points != nullptr) {
-    char* ep = strdup(points);
+std::vector<std::string> storeEndPoints(const std::string points) {
+  std::vector<std::string> endpointNames;
+  if (!points.empty()) {
+    char* ep = strdup(points.c_str());
```

Comment:
Maybe we can keep this all as `std::string`?

Reply:
Sure